dac_calibration_server.py

```diff
@@ -349,7 +349,7 @@ def correct_analog(self, c, data, average_ends=False, dither=False):
 
     @setting(33,
         'Correct Analog FT',
-        data=['*v: Single channel data'],
+        data=['*c: Single channel data'],
```

Comment:
I'm inclined to leave this as `*c` for now, which is what it was before and also what we have on the IQ correction settings. If we really wanted to be pedantic about things, then we'd probably want `*c[ns]` for the units, since the fourier transformed data technically has dimensions of time (that's what the envelope code spits out when you evaluate in fourier space, for example), but I don't really want to go there now.